wire-ios-data-model/sourcery/generated/AutoMockable.generated.swift

```diff
@@ -0,0 +1,35 @@
+// Generated using Sourcery 1.8.1 — https://github.com/krzysztofzablocki/Sourcery
+// DO NOT EDIT
+// swiftlint:disable line_length
+// swiftlint:disable variable_name
+
+import Foundation
+#if os(iOS) || os(tvOS) || os(watchOS)
+import UIKit
+#elseif os(OSX)
+import AppKit
+#endif
+
+
+@testable import WireDataModel
```

Comment:
I don't really get why this is empty / added to the test target

Reply:
This is where the generated mocks will end up. It's empty because I haven't generated any mocks yet, and it's in the test target so that we can start using the mocks in tests.

Reply:
ah ok it's all in one file